### PR TITLE
[FIX] Placing collectables doesn't cancel on hashchange

### PR DESCRIPTION
**Pull request by therocco**
# Description

The place collectables UI can stay visible when navigation back to other islands.

Steps to reproduce:

1. Navigate to Helios / Treasure Island
2. Navigate to home island
3. Open chest/buildings tab and try placing a placeable
4. Press the back button on mobile (or equivalent in PC maybe)
5. The UI is visible as shown in the screenshot.  If the ✅ button is enabled and you clicked it, it actually places the item in your home island even though you are not on that island while performing that action

Screenshot_20230330-123114

Fixes #issue
https://discord.com/channels/880987707214544966/1091086805295693834

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Testing in Chrome and Firefox manually triggering hashchange events and using each browsers navigation buttons and shortcuts. Tested via mobile using Coinbase Wallet and pinging my local dev environment.

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/farming/hud/components/PlaceableController.tsx [hunk 1]
```diff
@@ -72,6 +72,28 @@ export const PlaceableController: React.FC = () => {
...
 
+  const handleLocationHashChange = () => {
+    const locationHashLength = window.location.hash.split("/").length;
+    const isEditingActive =
+      child.state.matches({ editing: "idle" }) ||
+      child.state.matches({ editing: "dragging" });
+
+    // checks to see if editing is active and if the current map
+    // location is anything other than default "Home" route
+    if (
```
[the remark this answers is not in the document]
Comment (spencerdezartsmith) [reply src/features/farming/hud/components/PlaceableController.tsx:83]: No worries on the late reply!

Yep I understand that but what I was saying is that I don't think we need to do the check to see if we're in the `editing` state. If the `PlaceableController` is open then we are in the `editing` state by default.

[the remark this answers is not in the document]
Comment (therocco) [reply src/features/farming/hud/components/PlaceableController.tsx:83]: Gotcha. I can take a look at it again after dinner! Thanks for the feedback!

**Comment by AzaroQL** [comment]
Cant be reproduced on latest main branch.

**Comment by spencerdezartsmith** [comment]
I am going to close this as it doesn't appear to still exist.

**Pull request closed without merging**